### ISSUE TITLE
weights about dataset

### ISSUE THREAD

**Issue by NanWangAC** [zanilzanzan/FuseNet_PyTorch#16]
Hi, I have some questions about how do you calculate category weights for nyudv2 datasets and sun datasets?

**Comment by zanilzanzan** [comment]
I didn’t use class mapping for NYUv2 during training. However, if you want to incorporate that to cross-entropy loss, in order to calculate these weights, you can sum all the pixels belonging to each class in the training set and divide them by the total sum of all the pixels for all classes. Ignore zero labels.

**Comment by zanilzanzan** [comment]
Please let me know when your problem is solved, so I can close this issue.

**Comment by NanWangAC** [comment]
Ok, I get it! But the weights in **_weights.txt_** doesn't seem to calculate that way

**Comment by hazirbas** [comment]
Hi,
please divide the number pixels by the number of images on which the class present, and then inverse the frequency as in:

`https://arxiv.org/abs/1411.4734 `

**Comment by NanWangAC** [comment]
I get it! thanks for ur kind! (#^.^#)
